Task3_Products/schemas.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel, conint, condecimal
+
+
+class CategoryBase(BaseModel):
+    name: str
+    parent_id: Optional[int] = None
+
+
+class CategoryCreate(CategoryBase):
+    pass
+
+
+class CategoryUpdate(BaseModel):
+    name: Optional[str] = None
+    parent_id: Optional[int] = None
+
+
+class Category(CategoryBase):
+    id: int
+
+
+class ProductBase(BaseModel):
+    name: str
+    description: Optional[str] = None
+    price: condecimal(ge=0, decimal_places=2)
+    quantity: conint(ge=0)
```

Comment:
Price and quantity can't be zero, db fails in this case

Reply:
I agree about the price, but the quantity may be zero.